File: gestorProcessJudiciais/gestorProcessJudiciais/Models/ProcessosJudiciais.cs
namespace gestorProcessJudiciais.Models
{
    public class ProcessosJudiciais
    {
        public long Id { get; set; }
        public string NProcesso { get; set; }
        public string NomeDoProcesso { get; set; }
        public string Descricao { get; set; }
        public int Caixa { get; set; }
        public string Usuario { get; set; }
    }
}

File: gestorProcessJudiciais/frontGestor/src/app/Models/ProcessosJudiciais.ts
export class ProcessosJudiciais {
    id: number;
    nProcesso: string;
    nomeDoProcesso: string;
    descricao: string;
    caixa: number;
    usuario: string;
  
    constructor() {
      this.id = 0;
      this.nProcesso = '';
      this.nomeDoProcesso = '';
      this.descricao = '';
      this.caixa = 0;
      this.usuario = '';
    }
  }
  